docs/Modding/Tilesets/graphics.rst

```diff
@@ -0,0 +1,446 @@
+Tile Specification Files
+************************
+
+Creating a Freeciv21 Tileset requires a mix of editing plain text :file:`spec` files as well as creating
+graphics files saved in the :file:`.png` image format. When complete, a Tileset editor will have created
+the canvas that Freeciv21 players play upon. Next to :doc:`../Rulesets/index`, creating Tilesets is one of the
+most important aspects of a Freeciv21 game.
+
+Using Graphics
+---------------
+
+To use different graphics with Freeciv21, use the :code:`--tiles` argument to the Freeciv21 client. Eg, to use
+the 'engels' graphics, start the client as:
+
+.. code-block:: rst
+
+    freeciv21-client --tiles engels
+
+
+What Freeciv21 actually does in this case is look for a file called :file:`engels.tilespec` somewhere in your
+Freeciv21 data path. That :file:`.tilespec` file contains information telling Freeciv21 which graphics files
+to use, and what those graphics files contain.
+
+That is all you need to know to use alternative graphics provided by Freeciv21 or by third-party add-ons. The
+rest of this file describes, though not fully, the contents of the :file:`.tilespec` file and related files.
+This is intended as developer reference, and for people wanting to create/compile alternative tilesets and
+modpacks for Freeciv21.
+
+Overview
+--------
+
+The purpose of the :file:`.tilespec` file and related :file:`.spec` files is to allow the detailed layout of
+the graphics within the files to be flexible and not hard-coded into Freeciv21, and to allow add-ons to
+conveniently provide additional graphics.
+
+There are two layers to the :file:`.tilespec` files:
+
+* The top-level file is named, for example, :file:`trident.tilespec`. The basename of this file (here,
+  'trident') corresponds to the parameter of the :code:`--tiles` command-line argument for the Freeciv21 client,
+  as described above.
+
+* The top-level tilespec file contains general information on the full tileset, and a list of files which
+  specify information about the individual graphics files. These filenames must be located somewhere in the
+  data path, though not necessarily the same place as the top-level tilespec file. Note that with this system
+  the number and contents of the referenced files are completely flexible at this level.
+
+An exception is that the intro graphics must be in individual files, as listed in the tilespec file, because
+Freeciv21 treats these specially: these graphics are freed after the game starts, and reloaded later as
+necessary.
+
+Graphics Formats
+----------------
+
+All clients currently use 24 or 32 bit PNGs.
```

Comment:
Wait, there are different clients? Any file format supported by Qt can be used, but PNG is the most widely used and supported. There are no restrictions on the details of the PNG files (that I know of).